<launch>
    <node pkg="drivetrain" type="goto_pt.py" name="goto_pt" output="screen" >

        <param name="topic_goal_in" type="string" value="/plan_executor/goal_out"/>
        <param name="topic_drive_out" type="string" value="vesc/ackermann_cmd_mux/input/nav3"/>
        <param name="topic_plan_req" type="string" value="goto_pt/plan_req"/>
        <param name="base_frame" type="string" value="base_link"/>
        <param name="map_frame" type="string" value="odom"/>

        <param name="k_steer" type="double" value="0.3"/>
        <param name="k_dist" type="double" value="4.0"/>
        <param name="max_vel" type="double" value="2"/>
        <param name="max_theta" type="double" value="0.3"/>
        <param name="threshold" type="double" value="0.2"/>
        <param name="rate" type="double" value="30"/>
    </node>
    <node pkg="drivetrain" type="plan_executor.py" name="plan_exec" output="screen" >

        <param name="topic_plan_in" type="string" value="global_plan"/>
        <param name="topic_req_in" type="string" value="goto_pt/plan_req"/>
        <param name="topic_output" type="string" value="plan_executor/goal_out"/>
    </node>


    <node pkg="navstack" type="backup_recovery.py" name="backup_recovery" output="screen"/>


</launch>
